oe_content.module

```diff
@@ -137,5 +137,13 @@ function oe_content_node_prepare_form(EntityInterface $entity, $operation, FormS
 
   if (!empty($content_owners)) {
```

Comment:
I think this check is not needed since we already assert $site_information->hasDefaultContentOwners() before, but check it out just in case.